src/Zicht/Bundle/PageBundle/Admin/PageAdmin.php

```diff
@@ -10,7 +10,7 @@
 use Sonata\AdminBundle\Datagrid\ListMapper;
 use Sonata\AdminBundle\Form\FormMapper;
 use Sonata\AdminBundle\Show\ShowMapper;
-use Sonata\Form\Type\CollectionType as SonataCollectionType;
+use Sonata\CoreBundle\Form\Type\CollectionType;
```

Comment:
Dat is vreemd. De Sonata "core" bundle heeft Sonata juist weg gedaan en opgesplitst in o.a. een Sonata Form bundle. Ik zou de aanpassing juist andersom verwachten 🤔

Reply:
Snap ik. Dit fixt het inderdaad in ieder geval en dan werkt het weer. Maar wel apart 😄

Reply:
Alleen ben ik bang dat het dan weer in andere projecten stuk gaat...

Reply:
Hmm, ik zou in het project kunnen kijken of ik hem vast kan zetten op een oudere versie. Of anders binnen het project deze file extenden en overrulen?

Reply:
Of in het project niet `sonata-project/core-bundle` gebruiken

Reply:
@7ochem Je voorstel werkt daarom sluit ik deze pull request! Thanks voor het meedenken :balloon: 

Hier is de pull request: https://github.com/zicht/tinbergen.nl/pull/192